app/main.py

```diff
@@ -1,6 +1,30 @@
+import os
+import requests
+
+URL = "https://api.weatherapi.com/v1/current.json"
+
+API_KEY = os.getenv("API_KEY")
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    response = requests.get(
+        URL,
+        params={
+            "key": API_KEY,
+            "q": "Paris"
+        }
+    )
+
+    weather_data = response.json()
+    localtime = weather_data["location"]["localtime"]
+    temp_c = weather_data["current"]["temp_c"]
+    condition = weather_data["current"]["condition"]["text"]
+    print("Performing request to Weather API for city Paris...")
+    print(
+        f"Paris/France {localtime} \n"
+        f"Weather: {temp_c} Celsius, \n"
+        f"{condition}"
+    )
```

Comment:
Firstly, you need to check that response status is `200`, And only after that you can show information to user